azure-pipelines.yml

```diff
@@ -1,8 +1,9 @@
 variables:
-  buildNumber: $[ counter('constant', 13000) ] # Start higher than our AppVeyor versions. Every build (pr or branch) will increment.
+  versionNumber: '3.1.0'  # Minor version will be incremented by automation
```

Comment:
>'3.1.0 [](http://example.com/codeflow?start=17&length=6)

@brettsam / @fabiocav Should this read '3.x.0' or something ? Or is it okay to add instructions to update both common.props and the yml ?

Note: Automation relies on this today.

Reply:
Scratch that. Looks like the dynamic name updated I tried in iteration 3 started is working now... Here's a sample build with the same https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=16285&view=results

So, with the latest iteration, only update to common.props is needed and the above comment is invalid. @brettsam maybe you can use this for v4.x name as well ?

Reply:
yes!